### PR TITLE
feat: Add boskonf-banner

### LINKED ISSUE #47: Add big thing on frontpage for Bergen Open Source Konferanse
Add a big thing promoting our conferance on the frontpage of fribyte.no. Should be a button redirecting boskonf.no

### PR DESCRIPTION
**Pull request by johlys**
Added a banner for the conferanse to index.html.
banner_index

Closes #47

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -7,10 +7,33 @@
...
 {% block content %}
   <section class="hero">
-    <img class="illustration" src="{{ get_url(path="/img/illustrations/undraw_server.svg") | safe }}" alt="">
-
+    <!-- Old website layout. Revert when conferance is done -->
+    <!-- <img class="illustration" src="{{ get_url(path="/img/illustrations/undraw_server.svg") | safe }}" alt="">
     <p>
       {{ config.description }}
+    </p> -->
+    <!-- <img class="illustration" src="{{ get_url(path="/img/boskonf_banner.svg") | safe }}" alt=""> -->
```
Comment (timharek) [inline templates/index.html:15]: suggestion: I guess there isn't any need to comment this part out as we can revert the PR 😊

File: templates/index.html [hunk 2]
```diff
@@ -7,10 +7,33 @@
...
+        background: url({{get_url(path="/img/boskonf_banner.svg") | safe }}) center / cover no-repeat;
+        background-size: contain;
+      }
+    </style>
+
+    <p>
+      Velkommen til <a href="https://boskonf.no/">Bergen Open Source</a>, arrangert av friByte! Utforsk åpen kildekode gjennom innsiktsfulle foredrag fra erfarne IT-utviklere og lidenskapelige entusiaster. Bli med og oppdag kraften av åpen kildekode i hjertet av Bergen!
     </p>
     <div class="button-container center">
       <a href="{{ get_url(path="@/bli-medlem.md") }}" class="btn primary center underscore">{{ trans(key="bli_medlem", lang=lang) }}</a>
```
Comment (timharek) [inline templates/index.html:39]: suggestion: Should we replace the "Bli medlem"-button with a link to the boskonf-website? 💭

**Review by timharek — CHANGES_REQUESTED** [review]
Nice! ☺️ I only had some tiny suggestions 😇

File: templates/index.html (hunk 1 above)
Comment (mathiash98) [reply templates/index.html:15]: thats true

File: templates/index.html (hunk 2 above)
Comment (mathiash98) [reply templates/index.html:39]: good idea

**Review by SindreKjelsrud — APPROVED** [review]

**Review by SindreKjelsrud — APPROVED** [review]

**Review by SindreKjelsrud — APPROVED** [review]

**Review by mathiash98 — APPROVED** [review]

**Pull request merged**